domain/consensus/model/interface_processes_ghostdagmanager.go

```diff
@@ -6,4 +6,6 @@ import "github.com/kaspanet/kaspad/domain/consensus/model/externalapi"
 type GHOSTDAGManager interface {
 	GHOSTDAG(blockHash *externalapi.DomainHash) error
 	ChooseSelectedParent(blockHashA *externalapi.DomainHash, blockHashB *externalapi.DomainHash) (*externalapi.DomainHash, error)
+	Less(blockHashA *externalapi.DomainHash, ghostdagDataA *BlockGHOSTDAGData,
```

Comment:
I think less makes more sense, because this one returns a bool.
Compare returns an int, Less returns bool. 
Less is an idiomatic name in Go.
Only reason why this is no standard less is because it gets 2 params per compared thingy, but that is true for Compare as well.